src/licenses/bsd_3_clause.toml

```diff
@@ -0,0 +1,41 @@
+name = "bsd-3-clause"
+description = "Quelques informations à propos de la license \"The 3-Clause BSD License\"."
+
+[[embeds]]
+title = "The 3-Clause BSD License"
+thumbnail = "https://opensource.org/files/OSI_Approved_License.png"
+description = """
+Aussi connu sous les noms *"New BSD License"* ou *"Modified BSD License"*, la license BSD
```

Comment:
While I do acknowledge these translations, I am pretty sure that using the pseudo-official alternative names in English is better since they are more used across the web